### ISSUE TITLE
[STORY] User troubleshooting a failing pipeline

### ISSUE THREAD

**Issue by mrusyd** [Australian-Imaging-Service/pipelines#155]
### Links

Epic: #
Feature: #78 

### Description

@mrusyd: Story on the board reads: "As a pipeline user, I'm finding that a pipeline is failing and it is not clear why". Another story on the board that belongs to the same cluster reads: "As a user, I would like to see a list of problems that other users have had with a particular pipeline", and that seems to be a start of a troubleshooting process.

The first sentence is a problem statement, and the second talks about a step that seems have a goal of starting the troubleshooting. We have to work on these and write one or more user and enabler stories. This one is the first, and it is not ready as i stands when it was created on 19/9/2022. What exactly do we want to do to address this problem of a failing pipeline? 

Maybe the first go at this story should be: "As a pipeline user I want to troubleshoot a failing pipeline so that I can decide what to do next"? Maybe we can get into the details sufficiently by writing detailed acceptance criteria, with a risk of the story becoming too big. It is a start.

### Acceptance Criteria

> Acceptance criteria is used to confirm that the story was completed correctly. Criteria can be written in any format, or a "given-when-then" form may be used:

- [ ] 1. [ACCEPTANCE CRITERIA]

### Blocked by

- [ ]
